### PR TITLE
New, add kernel args to nodes per OSDOCS-479 Jira

### PR DESCRIPTION
**Pull request by chrisnegus**
This is a new procedure for adding kernel arguments to OpenShift 4.2 nodes using machineconfigs.

### CODE REVIEW

File: modules/nodes-nodes-kernel-arguments.adoc [hunk 1]
```diff
@@ -0,0 +1,146 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-nodes-working.adoc
+
+[id="nodes-nodes-kernel-arguments_{context}"]
+= Adding kernel arguments to Nodes
+
+In some special cases, you may want to add kernel arguments
```
Comment (ahardin-rh) [inline modules/nodes-nodes-kernel-arguments.adoc]: s/may/might (per IBM Style Guide)

File: modules/nodes-nodes-kernel-arguments.adoc [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+[WARNING]
+====
+Improper use of kernel arguments can result in your systems becoming unbootable.
+====
+
+Examples of kernel arguments you could set include:
+
+* **selinux=0**: Disables Security Enhanced Linux (SELinux).
+While not recommended for production, disabling SELinux can
+improve performance by 2%-3%.
```
Comment (ahardin-rh) [inline modules/nodes-nodes-kernel-arguments.adoc]: 2% - 3%  (spacing around the -)

File: modules/nodes-nodes-kernel-arguments.adoc [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+You could consider `nosmt` in multi-tenant environments to reduce
+risks from potential cross-thread attacks. By disabling SMT, you essentially choose security over performance.
+
+See link:https://www.kernel.org/doc/Documentation/admin-guide/kernel-parameters.txt[Kernel.org kernel parameters]
+for a list and descriptions of kernel arguments.
+
+In the following procedure, you create a MachineConfig that identifies:
+
+* A set of machines to which you want to add the kernel argument
+(in this case, machines with a worker role).
```
Comment (ahardin-rh) [inline modules/nodes-nodes-kernel-arguments.adoc]: argument. In this case, the machines have a worker role.

File: modules/nodes-nodes-kernel-arguments.adoc [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+  name: 05-worker-kernelarg-selinuxoff<2>
+spec:
+  config:
+    ignition:
+      version: 2.2.0
+  kernelArguments:
+    - selinux=0<3>
+----
++
+<1> Applies the new kernel argument only to worker nodes
```
Comment (ahardin-rh) [inline modules/nodes-nodes-kernel-arguments.adoc]: Period after each description

**Comment by ahardin-rh** [comment]
@chrisnegus This looks great! I had just a few minor comments. 

Also, I see that you have this submitted against the enterprise-4.2 branch. We'll also need to submit this against master. We can either submit a new PR against master, merge, and then cherry-pick it to the enterprise-4.2 branch. Alternatively, we can merge this once you make the latest changes and squash the commits into one, and then submit to the master branch in a new PR. Let me know if I can help with that.

**Comment by vikram-redhat** [comment]
@miabbott reviewed this from QE.

**Pull request merged**